Build fails for mjolnir

Trying to run `docker build` I get:

```
configure: error: Could not find valhalla_skadi!
The command '/bin/sh -c cd mjolnir && ./autogen.sh && ./configure CPPFLAGS=-DBOOST_SPIRIT_THREADSAFE && make -j4 && make install && cd ..' returned a non-zero code: 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -58,6 +58,9 @@ RUN cd baldr && ./autogen.sh && ./configure CPPFLAGS=-DBOOST_SPIRIT_THREADSAFE &
 ADD sif sif
 RUN cd sif && ./autogen.sh && ./configure CPPFLAGS=-DBOOST_SPIRIT_THREADSAFE && make -j4 && make install && cd ..
 
+ADD skadi skadi
+RUN cd skadi && ./autogen.sh && ./configure CPPFLAGS=-DBOOST_SPIRIT_THREADSAFE && make -j4 && make install && cd ..
+
 ADD mjolnir mjolnir
 RUN cd mjolnir && ./autogen.sh && ./configure CPPFLAGS=-DBOOST_SPIRIT_THREADSAFE && make -j4 && make install && cd ..
 

--- a/fetch-repos.sh
+++ b/fetch-repos.sh
@@ -1,5 +1,5 @@
 #!/bin/sh
-for repo in midgard baldr sif mjolnir loki odin thor tyr; do
+for repo in midgard baldr sif skadi mjolnir loki odin thor tyr; do
   if [ ! -d $repo ]
   then
     git clone --recurse-submodules https://github.com/valhalla/$repo.git

